Bai 4/Bai 4.csproj

```
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <RootNamespace>Bai_4</RootNamespace>
```

```suggestion
    <RootNamespace>Bai4</RootNamespace>
```
vừa a thiếu. đổi tên folder bỏ dấu cách đi nhé